src/main/java/com/skeleton/feed/entity/Statistics.java

```diff
@@ -0,0 +1,49 @@
+package com.skeleton.feed.entity;
+
+import com.skeleton.user.entity.User;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import java.util.Date;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import jakarta.persistence.Id;
+
+@Entity
+@Getter
+@Table(name = "post_statistics")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Statistics {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User userId;
+    @OneToMany(mappedBy = "statistics")
```

Comment:
```suggestion
    @OneToOne
```
```
Caused by: org.hibernate.AnnotationException: Property 'com.skeleton.feed.entity.Statistics.post' is not a collection and may not be a '@OneToMany', '@ManyToMany', or '@ElementCollection'
```

서윤님 지금 Statistics와 Post의 연관 관계 어노테이션이 잘못 설정되어 있어서 어플리케이션 실행 시 위와같은 오류가 발생합니다. PR 하시기 전에는 애플리케이션이 동작하는 지 확인 하신 후 PR 해주시면 감사하겠습니다. 😂